WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -553,6 +554,16 @@ def my_competitions
         :competitor_limit_reason,
         :remarks,
         :registration_requirements,
+        :free_competition_enabled,
+        :on_the_spot_registration_enabled,
+        :on_the_spot_entry_fee,
+        :on_the_spot_entry_fee_lowest_denomination,
+        :refund_policy,
+        :refund_policy_amount,
+        :refund_policy_limit_date,
+        :guests_entry_fee_enabled,
```

Comment:
As we discussed offline, I don't think any of these `*_enabled` fields need to make it to the backend. I think they should just be frontend checkboxes that determine if certain other fields are visible.

Reply:
I will look at how to do that correctly.